packager/media/event/muxer_listener_factory.cc

```diff
@@ -81,10 +82,12 @@ std::list<std::unique_ptr<MuxerListener>> CreateHlsListenersInternal(
 
 MuxerListenerFactory::MuxerListenerFactory(bool output_media_info,
                                            MpdNotifier* mpd_notifier,
-                                           hls::HlsNotifier* hls_notifier)
+                                           hls::HlsNotifier* hls_notifier,
+                                           bool use_segment_list)
```

Comment:
Can you move it after line 83 for style consistency (input parameters before output parameters)?